lib/optcarrot/config.rb

```diff
@@ -45,14 +45,15 @@ class Config
       },
       profiling: {
         print_fps: { type: :switch, desc: "print fps of last 10 frames", default: false },
+        print_p95fps: { type: :switch, desc: "print 95th percentile fps", default: false },
         print_fps_history: { type: :switch, desc: "print all fps values for each frame", default: false },
         print_video_checksum: { type: :switch, desc: "print checksum of the last video output", default: false },
         stackprof: { shortcut: "--stackprof-mode=cpu", aliases: :p },
         stackprof_mode: { type: "MODE", desc: "run under stackprof", default: nil },
         stackprof_output: { type: "FILE", desc: "stackprof output file", default: "stackprof-MODE.dump" }
       },
       misc: {
-        benchmark: { shortcut: %w(--headless --print-fps --print-video-checksum --frames 180), aliases: :b },
+        benchmark: { shortcut: %w(--headless --print-fps --print-p95fps --print-video-checksum --frames 180), aliases: :b },
```

Comment:
I'm reluctant against enabling fps history for `--benchmark` for two reasons: (1) it may change the performance charactaristics (maybe neglectable for short frames but I'm unsure) and (2) it is incompatible as the output of `--benchmark` is parsed by some scripts.